docs/OrganizingCommittee/Committee-formation.md

```diff
@@ -0,0 +1,84 @@
+# Committee Formation
+
+Usually a hack week is spearheaded by one or several individual who have the motivation to host a hack week for a specific community. Once topic of a hackweek is formulated, the process of gathering together other committee memebers should begin, ideally 8 - 12 months prior to the event. 
+
+Below we list the typical roles across an organizing committee. We note that for some events, certain roles might not be considered an integral part of the formal organizing committee, for example the tutorial leads. However, we believe there are several advantages to including these people at the decision making level of the overall event. These include an increased sense of ownership that results in organizers putting forth their best effort in designing and delivering content, and a broadening of the collective wisdom of the organizing committee around important decisions due to the presence of a diversity of perspectives. Early career faculty and researchers also often report that being recognized at the organizational level helps their resumes when it comes to applying for jobs. 

```

Comment:
I don't know if this is worth mentioning, but having too many people in the committee can also make things much harder? Astro Hack Week has had everything from one person to twelve, and I think my happy medium is 4-5. For the last AHW, where we had twelve organizers, we ended up in a situation where it was basically impossible to schedule any telecons (also because were spread out across five time zones spanning ten hours in time difference), and too many people with opinions can lead to discussions that go on forever. At the very least, large committees might need some additional careful facilitating (i.e. someone being in the position to shorten discussions + making sure important decisions get made)